mmv1/third_party/terraform/services/compute/resource_compute_network_firewall_policy_rule_test.go.tmpl

```diff
@@ -149,6 +147,187 @@ func TestAccComputeNetworkFirewallPolicyRule_securityProfileGroup_update(t *test
   })
 }
 
+func TestAccComputeNetworkFirewallPolicyRule_secureTags(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"org_id":        envvar.GetTestOrgFromEnv(t),
+		"project_name":  envvar.GetTestProjectFromEnv(),
+		"service_acct":  envvar.GetTestServiceAccountFromEnv(t),
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckComputeNetworkFirewallPolicyRuleDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccComputeNetworkFirewallPolicyRule_secureTags(context),
+			},
+			{
+				ResourceName:      "google_compute_network_firewall_policy_rule.primary",
+				ImportState:       true,
+				ImportStateVerify: true,
+        // Referencing using ID causes import to fail
+				ImportStateVerifyIgnore: []string{"firewall_policy", "project"},
+			},
+			{
+				Config: testAccComputeNetworkFirewallPolicyRule_secureTagsUpdate(context),
```

Comment:
Please add `ConfigPlanCheck` to this step: [See "handwritten" section](https://googlecloudplatform.github.io/magic-modules/develop/test/test/#add-an-update-test)